contrib/babelfishpg_common/src/babelfishpg_common.h

```diff
@@ -70,5 +70,5 @@ typedef struct common_utility_plugin
 										  bool *isBaseDate, int *variantHeaderLen);
 	Oid			(*lookup_tsql_datatype_oid) (const char *typestr);
 	int32_t		(*GetUTF8CodePoint) (const unsigned char *in, int len, int *consumed_p);
-
```

Comment:
we need to make similar changes in pltsql.h also. Not sure why this changes are working in our testing

Reply:
Not required as definition of common_utility_plugin only exists in babelfishpg_common.h